libs/langgraph/langgraph/graph/state.py

```diff
@@ -760,12 +760,37 @@ def branch_writer(
                     config, cast(Sequence[Union[Send, ChannelWriteEntry]], writes)
                 )
 
+        # detect branch input schema
+        input = None
+
+        # detect input schema annotation in the branch callable
+        try:
+            if (
+                isinstance(branch.path, RunnableCallable)
```

Comment:
What if it's a RUnnableLambda?

Reply:
yea might need to check for that too just in case